CMake/FindNSS.cmake

```diff
@@ -10,6 +10,8 @@ set(NSS_LIBRARIES ${PC_NSS_LINK_LIBRARIES})
 set(NSS_INCLUDE_DIRS ${PC_NSS_INCLUDE_DIRS})
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(NSS DEFAULT_MSG NSS_INCLUDE_DIRS NSS_LIBRARIES)
+find_package_handle_standard_args(NSS
+    REQUIRED_VARS NSS_LIBRARIES NSS_INCLUDE_DIRS
+    VERSION_VAR PC_NSS_VERSION)
```

Comment:
`PC_NSS_VERSION` will not exist on platforms without PkgConfig.
I think that this is ok since CMakeLists.txt does not specify a version requirement, making this purely cosmetic (right?)

Reply:
correct, this will simply do nothing if the variable is not set.